saleor/tests/e2e/product/utils/preparing_product.py

```diff
@@ -59,3 +60,55 @@ def prepare_product(
     ]["amount"]
 
     return product_id, product_variant_id, product_variant_price
+
+
+def prepare_digital_product(
```

Comment:
Shouldn't it be added to __init__.py?

Reply:
I added `prepare_digital_product` and `prepare_product` to __init__.py but in this PR I will use `from ..product.utils import prepare_digital_product` notation only for this test and update the rest test in separate PR.